core/src/main/resources/reference.conf

```diff
@@ -562,8 +562,13 @@ datastax-java-driver {
         # the journal does not use any counters or collections
         default-idempotence = true
       }
-
-
+    }
+    akka-persistence-cassandra-snapshot-profile {
+      basic.request {
+        consistency = ONE
```

Comment:
this might look like a big change, but that is what we used in the pre 1.0 version https://github.com/akka/akka-persistence-cassandra/blob/release-0.x/core/src/main/resources/reference.conf#L546